docs/RDMA-example.md

```diff
@@ -0,0 +1,69 @@
+
+# RDMA example
+
+This guide provides example commands that can be used to send a video file via RDMA on a single node.
+
+## Prerequisites
+
+1. Media Communications Mesh is built
+```
+./build.sh
+```
+
+2. Libfabric is build and installed with RDMA capable providers
+
+    You can check if this is true with:
+    ```
+    fi_info -p verbs -v
```

Comment:
$

Reply:
added

Reply:
not really. so without or with "$" finally?

Reply:
I removed "$" everywhere after all. I think it was the consensus after we talked about it.